scripts/prepare_bootstrap_inputs.py

```
@@ -0,0 +1,294 @@
+        droplet_tag=resolved_droplet_tag,
+        state_file=state_file,
+        ca_certificate_path=ca_cert_path,
+        ssh_identity_path=ssh_identity,
+    )
+
+
+def _resolve_input(
+    param_value: str | Path | None,
+    env_key: str,
+    default: str | Path | None = None,
+    required: bool = False,
```

Ruff (0.14.6)

Boolean-typed positional argument in function definition

(FBT001)